SECP-service/src/main/resources/assets/app/views/portal/configure.html

```diff
@@ -0,0 +1,113 @@
+<body>
```

Comment:
If anyone has any suggestions to avoid the duplication here and in the controller let me know.

Reply:
Thanks man!